cli/cmd/cli_state.go

```diff
@@ -216,7 +216,7 @@ func (c *cliState) NewClient() error {
 
 // InteractiveMode returns true if the cli is running in interactive mode
 func (c *cliState) InteractiveMode() bool {
-	return !c.nonInteractive && !c.csvOutput
+	return !c.nonInteractive && !c.csvOutput && !c.jsonOutput
```

Comment:
This is kind of a different task than what this ticket intended. I think it's probably correct, but i'd maybe consider either adding more tests for this use case or moving it out of this PR.